samples/Infrastructure/Storage.fs

```diff
@@ -345,3 +346,25 @@ module Sql =
             let a = Arguments(p)
             let connection = connect log (a.ConnectionString, a.Schema, a.Credentials, a.AutoCreate) |> Async.RunSynchronously
             StorageConfig.Sql(SqlStreamStoreContext(connection, batchSize = a.BatchSize), cacheStrategy cache, unfolds)
+     module MsgDb =
```

Comment:
this should not be under `.Sql`, i.e. instead of `Storage.Sql.MsgDb`, this should be `Storage.MessageDb` (the `Sql` represents `SqlStreamStore`, not the fact that we're trying to lump SQL backed stores together in any way